src/spec.rs

```diff
@@ -93,8 +94,55 @@ impl ExtraFieldMagic {
     pub const ZIP64_EXTRA_FIELD_TAG: Self = Self::literal(0x0001);
 }
 
-/// This should be equal to `0xFFFFFFFF`.
+/// The number of bytes necessary to allocate a zip64 record for an individual file.
+///
+/// If a file larger than this threshold attempts to be written, and
```

Comment:
```suggestion
/// The file size at which a ZIP64 record becomes necessary.
///
/// If you attempt to write a file this size or larger, compressed or uncompressed, and
```
I believe a ZIP64 record is needed when the file is *exactly* this size, just to clarify that it's not strictly larger, and I recently changed the code to reflect that.

Reply:
Hmmm, so I just tried testing this, and it seems that at exactly this size the write does *not* fail. See diff:
```diff
diff --git a/src/spec.rs b/src/spec.rs
index 1148cd7ca..9525fef41 100755
--- a/src/spec.rs
+++ b/src/spec.rs
@@ -113,14 +113,11 @@ impl ExtraFieldMagic {
 /// // Writing an extremely large file for this test is faster without compression.
 /// let options = SimpleFileOptions::default().compression_method(zip::CompressionMethod::Stored);
 ///
-/// let big_len: usize = (zip::ZIP64_BYTES_THR as usize) + 1;
+/// let big_len: usize = zip::ZIP64_BYTES_THR as usize;
 /// let big_buf = vec![0u8; big_len];
 /// zip.start_file("zero.dat", options)?;
 /// // This is too big!
-/// let res = zip.write_all(&big_buf[..]).err().unwrap();
-/// assert_eq!(res.kind(), io::ErrorKind::Other);
-/// let description = format!("{}", &res);
-/// assert_eq!(description, "Large file option has not been set");
+/// let res = zip.write_all(&big_buf[..]).unwrap();
 /// // Attempting to write anything further to the same zip will still succeed, but the previous
 /// // failing entry has been removed.
 /// zip.start_file("one.dat", options)?;
```

Just changing `big_len` to equal `ZIP64_BYTES_THR` causes the doctest to fail, because the `.err().unwrap()` call fails. When I change `.err().unwrap()` to just `.unwrap()` and remove the checking of the error description (as in the diff), the zip is successfully created, with both file entries. Do you think this is wrong?

I have made the other changes you suggested to the docstring, however!